source/GameData.cpp

```diff
@@ -492,6 +492,15 @@ void GameData::UpdateSystems()
 
 
 
+
+void GameData::UpdateSystem(const std::string &name, const Ship *ship)
```

Comment:
maybe ugly?

Reply:
I'm okay with it not saying flagship explicitely but it should be a reference given we cant do anything without it